src/linechart/VoronoiCircleContainer.jsx

```diff
@@ -0,0 +1,76 @@
+'use strict';
+
+var React = require('react');
+var d3 = require('d3');
+var shade = require('../utils').shade;
+var VoronoiCircle = require('./VoronoiCircle');
+
+module.exports = React.createClass({
+
+  displayName: 'VornoiCircleContainer',
+
+  getDefaultProps() {
+    return { 
+      circleRadius: 3,
+      circleFill: '#1f77b4',
+      hoverAnimation: true
+    };
+  },
+
+  getInitialState() {
+    return { 
+      circleRadius: this.props.circleRadius,
+      circleFill: this.props.circleFill
+    }
+
+  },
+
```

Comment:
This suffers from the same bug that I had previously fixed. You need to also use

```javascript
componentWillReceiveProps(nextProps) {
   this.setState({
      circleRadius: nextProps.circleRadius,
      circleColor:  nextProps.circleFill
   });
},
```

Otherwise it won't update properly. See #211